### DISCUSSION TITLE
How to add monitoring for linkerd?

### DISCUSSION THREAD

**Discussion by Phoebus888** [linkerd/linkerd2#7295] [Q&A]
![image](https://user-images.githubusercontent.com/45359605/141954197-bf9cc116-fddd-4604-867b-d2e540510d37.png)
This format

**Comment by wmorgan** [comment]
Hi @Phoebus888. That looks like a screenshot of [Buoyant Cloud](https://buoyant.io/cloud)—that's one way to add a monitoring/alerting layer on top of Linkerd's metrics, but there are many options, including running something like [AlertManager](https://prometheus.io/docs/alerting/latest/alertmanager/) yourself. Can you tell us a bit more about what you're looking for?
